discord/channel.go

```diff
@@ -909,6 +1042,38 @@ func ApplyGuildIDToChannel(channel GuildChannel, guildID snowflake.Snowflake) Gu
 		c.guildID = guildID
 		return c
 	default:
-		panic("unknown channel type")
+		panic(fmt.Sprintf("invalid channel type: %d", channel.Type()))
```

Comment:
Is panicking the best idea here? It's always possible discord adds a new type and this should not break production bots. It should probably warn instead.

Reply:
hm yeah prob